backend/src/Altinn.AccessManagement.UI/Altinn.AccessManagement.UI/appsettings.AT21.json

```diff
@@ -9,7 +9,8 @@
   },
   "GeneralSettings": {
     "FrontendBaseUrl": "https://am.ui.at21.altinn.cloud",
-    "Hostname": "at21.altinn.cloud"
+    "Hostname": "at21.altinn.cloud",
+    "UseMockData": false
```

Comment:
Du trenger egentlig ikke å sette denne til false her i disse underfilene når samme setting er usatt eller satt til false i den generelle appsettings, men det gjør ikke noen skade.

Reply:
Da har deg og Jon Kjetil forskjellig mening: https://altinndevops.slack.com/archives/CQPNM75L3/p1692352916875959

Reply:
tror det er lurt som han sier for ja, hvis man får sett appsettings.json til true som skjedde ved forrige merge så vil disse overstyre den